drawer-view/DrawerView.swift

```diff
@@ -261,6 +270,12 @@ public class DrawerView: UIView {
         runningAnimators += [transitionAnimator]
     }
     
+    @objc private func drawerViewGestureTappedOut(recognizer: UITapGestureRecognizer) {
+        if closeOnTapOutside {
```

Comment:
When there is no `else` or `if else` statement(s), it's better to use `guard` statement, because it removes unneeded nesting levels and indicates the intention that  "`we should leave early, because if this condition is not true, then there is no point in the resining of the code block`". 

Please, don't take it personally, it's just a small improvement. By no means I'm perfect, in fact there is a couple of places in this project (the code that I wrote before), where the aforementioned logic needs to be applied. 😉

I will make the changes myself after merging your PR.

Reply:
No offense taken ! Thanks for the tip, forgot about that.